rllib/examples/autoregressive_action_dist.py

```diff
@@ -144,6 +144,9 @@ def get_cli_args():
         .training(gamma=0.5)
         # Use GPUs iff `RLLIB_NUM_GPUS` env var set to > 0.
         .resources(num_gpus=int(os.environ.get("RLLIB_NUM_GPUS", "0")))
+        # Batch-norm models have not been migrated to the RL Module API yet.
+        .training(_enable_learner_api=False)
+        .rl_module(_enable_rl_module_api=False)
```

Comment:
link these two tests in the description of the issue, so the problem is clearly elaborated.